src/main/webapp/WEB-INF/themes/desktop/ftl/article/amendment.ftl

```diff
@@ -23,15 +23,14 @@
 <#include "citation.ftl" />
 <#list amendments as amendmentGroup>
   <div class="amendment amendment-${amendmentGroup.type.cssName} toc-section">
-  <#assign title>
-    <@pluralize count=amendmentGroup.amendments?size value=amendmentGroup.type.displayName />
-  </#assign>
+  <#assign title><#include "amendmentGroupTitle.ftl"></#assign>
+  <#assign pluralizedTitle><@pluralize count=amendmentGroup.amendments?size value=title?trim/></#assign>
```

Comment:
Much clearer and more scannable than before.

Reply:
lol. still pretty crazy though